frontend/public/components/namespace.jsx

```diff
@@ -124,149 +128,376 @@ const fetchNamespaceMetrics = () => {
   return Promise.all(promises).then((data) => _.assign({}, ...data));
 };
 
-const namespacesColumnClasses = [
-  '', // name
-  '', // status
-  classNames('pf-m-hidden', 'pf-m-visible-on-sm'), // labels
-  classNames('pf-m-hidden', 'pf-m-visible-on-lg pf-u-w-16-on-lg'), // created
-  Kebab.columnClass,
-];
+const projectAndNamespaceColumnInfo = Object.freeze({
```

Comment:
Let's just call this `namespaceColumnInfo`

Reply:
Done.